### PR TITLE
Feature/Integrate Database Session Object to Covid Score Functionality

### PR DESCRIPTION
**Pull request by danoand**
This PR stands up the covid score functionality to use the `psycopg2` database adapter.  This PR:

* moves the state population dictionary into a standalone file: `constants.py`
* move the covid score functions into a standalone file: `covid_score.py`
* update the `predict.py` file to use the database session class

This PR is based on the changes in PR #15 : https://github.com/Lambda-School-Labs/LabsPT13-Resfeber-C-DS/pull/15

### CODE REVIEW

File: project/app/api/covid_score.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        "color":    the state's covid score as a color ("green", "yellow", "red")
+        "data":     CDC covid API data
+        "error":    error message (if applicable)
+    """
+    # Define a return object
+    ret_dict = {
+        "ok":    False,
+        "score": None,
+        "delta": None,
+        "color": None,
```
Comment (AmyBeisel) [inline project/app/api/covid_score.py:91]: Color would be awesome to have this be returned.  Red/yellow/green.

File: project/app/api/covid_score.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    
+    # Return results
+    ret_dict["ok"]       = True
+    ret_dict["error"]    = None
+    ret_dict["new_case"] = cvd_dict["new_case"]
+    ret_dict["data"]     = cvd_dict["data"]
+    return ret_dict
+
+# Generate a Covid "score" for a given state
+def gen_covid_score(db_conn, ste):
```
Comment (unburied) [inline project/app/api/covid_score.py:69]: Great work get the calculations for the covid flags. Would suggest breaking this function into smaller, more modular subfunctions for ease of testing/updating.

File: project/app/api/covid_score.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    ret_dict["error"] = None
+      
+    # Assign the most recent CDC Covid API data
+    ret_dict["data"] = {"start": intv_start_ncases["data"], "end": intv_end_ncases["data"]}
+
+    # Return 
+    return ret_dict
+
+# Generate score calculations for every state
+def calc_covid_deltas(db_conn):
```
Comment (unburied) [inline project/app/api/covid_score.py:176]: This is great for allowing the map to make more gradient colors

**Review by AmyBeisel — APPROVED** [review]
Thanks for explaining this code Dan. The color for green/yellow/red score plus having the map as a feature, this would be so helpful to the user!  Really nice work!

**Review by unburied — APPROVED** [review]
Clean code. Just need more atomic commits

**Review by richardOlson — APPROVED** [review]
Looks good.  Like the way the score is build.

**Pull request merged**